lib/puppet/provider/certs_bootstrap_rpm/katello_ssl_tool.rb

```diff
@@ -43,7 +43,8 @@ def last_rpm
       {'release' => release, 'rpm' => rpm}
     end
 
-    rpms.sort { |a,b| a['release'].to_i <=> b['release'].to_i }.last['rpm']
+    rpm = rpms.sort { |a,b| a['release'].to_i <=> b['release'].to_i }.last
+    rpm['rpm'] if rpm
```

Comment:
I think this will cause an error above - when we make the symlink on L25.  Probably need to gate that with a  `last_rpm.blank?` check